com.unity.ml-agents/Runtime/Analytics/Events.cs

```diff
@@ -48,6 +49,35 @@ public static EventActionSpec FromActionSpec(ActionSpec actionSpec)
         }
     }
 
+    /// <summary>
+    /// Information about an actuator.
+    /// </summary>
+    [Serializable]
+    internal struct EventActuatorInfo
+    {
+        public int BuiltInActuatorType;
+        public int NumContinuousActions;
```

Comment:
I'm on the fence about whether to include NumContinuousActions and NumDiscreteActions since we already have them in EventActionSpec - the EventActionSpec ones should be the sum of the values in the EventActuatorInfo list.

Reply:
if it doesn't hurt, I don't see a problem collecting the info.